singlestoredb/functions/signature.py

```diff
@@ -45,10 +45,8 @@
         np.uint8: 'uint8',
         np.longlong: 'uint64',
         np.ulonglong: 'uint64',
-        np.unicode_: 'str',
         np.str_: 'str',
         np.bytes_: 'bytes',
-        np.float_: 'float64',
```

Comment:
@kesmit13 Had to remove these due to changes in numpy.

Reply:
We can't drop support for older versions of numpy, so these will have to remain if they do exist in the installed version of numpy. I'll make a change in the main branch for this today that you can rebase in.